TYPO3.Neos/Resources/Private/Styles/Media/_Media.scss

```diff
@@ -597,12 +597,11 @@
 				}
 
 				.neos-button-small {
-					height: 32px;
-					width: 32px;
-					padding: 0 10px;
+					height: $wideMargin;
+					width: $wideMargin;
```

Comment:
Thanks for explaining, started reviewing that one already (will test it in detail tomorrow).